### PR TITLE
Add style to the page

### PR DESCRIPTION
**Pull request by MoRaad97**
### In this PR:
- Update and add elements to the page.
- Add style to the page using plain CSS.

### CODE REVIEW

File: src/template.html [hunk 1]
```diff
@@ -18,13 +18,14 @@ <h2>Recent Scores</h2>
...
         </div>
         <!-- Add Score -->
-        <div class="Add Your Score">
+        <div class="add-score">
           <h2>Add Your Score</h2>
           <form action="#" id="myForm">
-            <input required id="name" type="text " placeholder="Your Name" />
-            <input required id="score" type="range " placeholder="Your Score" />
+            <input class="input" required id="name" type="text " placeholder="Your Name" />
+            <input class="input" required id="score" type="range " placeholder="Your Score" />
```
Comment (tufoinnkuo10) [inline src/template.html:28]: - [ optional] it will be nice if you add some hover effects on your buttons

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @MoRaad97 

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ board updates automatically after refresh
✅ submit works well
✅ refresh works well

## Required Changes ♻️♻️
Check the comments under the review
- [ required] please give a good description of how the app works and functions in your readme file,  It should have instructions about where and how to generate any necessary keys, and credentials, how to set, store, and generate them, and how to start and run your app.

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @MoRaad97 , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ board updates automatically after refresh
✅ submit works well
✅ refresh works well

## Aspects to improve ♻️

- [x] As the previous Code Reviewer required. Please give a good description of how the app works and functions in your readme file, It should have instructions about where and how to generate any necessary keys, and credentials, how to set, store, and generate them, and how to start and run your app.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @MoRaad97, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations

- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Gitflow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title :heavy_check_mark:

> ## [Optional] suggestions:
> Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**